hollow/src/main/java/com/netflix/hollow/core/read/engine/list/HollowListDeltaHistoricalStateCreator.java

```diff
@@ -70,6 +70,12 @@ public void populateHistory() {
         }
     }
 
+    public void dereferenceTypeState() {
```

Comment:
A comment or java doc would be helpful here to understand why/where this method is required.

Reply:
Added